src/screens/CartScreen.tsx

```diff
@@ -0,0 +1,351 @@
+import * as React from 'react';
+import {
+  Image,
+  Button,
+  View,
+  StyleSheet,
+  Text,
+  ActivityIndicator,
+  FlatList,
+} from 'react-native';
+import {useSelector, useDispatch} from 'react-redux';
+import * as Sentry from '@sentry/react-native';
+import Toast from 'react-native-toast-message';
+import LinearGradient from 'react-native-linear-gradient';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import {RootState, AppDispatch} from '../reduxApp';
+import {selectImage} from './ToolStore';
+
+interface CartData {
+  sku: string;
+  name: string;
+  image: string;
+  id: number;
+  type: string;
+  price: number;
+  quantity: number;
+}
+interface subTotal {
+  quantity: number;
+  total: number;
+}
+export type UIToast = typeof Toast;
+
+const CartScreen = (props) => {
+  const dispatch = useDispatch();
+  const cartData = useSelector((state: RootState) => state.cart);
+  const [orderStatusUI, setOrderStatusUI] = React.useState(false);
+
+  const cartItems: Array<CartData> | [] = Object.values(cartData);
+  const computeCartTotal = (cartItems: Array<CartData>): subTotal => {
+    let aggregate = cartItems.reduce(
+      (acc, item) => {
+        acc.quantity += item.quantity;
+        acc.total += item.price;
+        return acc;
+      },
+      {total: 0, quantity: 0},
+    );
+    return aggregate;
+  };
+  const subTotalDisplay = (props: subTotal): React.ReactElement => {
+    const {quantity: q, total: t} = props;
+    const multiple = q > 1 ? 's' : '';
+    return (
+      <Text
+        style={{
+          marginTop: 20,
+          marginBottom: 20,
+          fontSize: 18,
+          fontWeight: '600',
+        }}>{`Subtotal (${q} item${multiple}): $${(t / 1000).toFixed(2)}`}</Text>
+    );
+  };
+  const performCheckoutOnServer = async () => {
+    // ----------- Sentry Start Transaction ------------------------
+    let transaction = Sentry.startTransaction({name: 'checkout'});
+    Sentry.configureScope((scope) => scope.setSpan(transaction));
+    // -------------------------------------------------------------
+
+    let data = await placeOrder(Toast);
+    // ----------- Sentry Finish Transaction -----------------------
+    const span = transaction.startChild({
+      data,
+      op: 'task',
+      description: `processing shopping cart result`,
+    });
+
+    span.finish();
+    transaction.finish();
+
+    // -------------------------------------------------------------
+  };
+  const placeOrder = async (
+    uiToast: null | UIToast = null,
+  ): Promise<Response> => {
+    setOrderStatusUI(true);
+    const data = {cart: Object.values(cartData)};
+    let response = await fetch(
+      'https://dustinbailey-flask-m3uuizd7iq-uc.a.run.app/checkout',
```

Comment:
could put a variable here. I can make this change if you prefer to merge first.

Reply:
you mean define it in a variable like `const url = https://<>` at the top of the file?
or could could put a `export const URL` in src/dsn.tsx where the dsn is defined as well.

If you think you can get that babel plugin working so we can use a .env file then try it. We have a babel.config.js but not a .babelrc like the npm module is recommending. @dgbailey